account_central_billing/models/res_partner.py

```diff
@@ -59,9 +58,7 @@ def get_billing_partner(self, vals, invoice=None):
         elif invoice:
             invoice_company = invoice.company_id
         else:
-            invoice_company = self.env["res.company"]._company_default_get(
-                "account.invoice"
-            )
+            invoice_company = self.env.company
```

Comment:
We are sure these are identical? I guess ultimately it is just a final catchall anyway and best we can do.